modules/ROOT/pages/syntax/keywords.adoc

```diff
@@ -0,0 +1,361 @@
+:description: This section contains a list of reserved keywords in Cypher.
+
+[[cypher-keywords]]
+= Keywords
+
+Keywords are words that have a special meaning in Cypher.
+
+The keywords are not recommended to be used as identifiers in the following contexts:
+
+* Variables
+* Labels
+* Relationship types
+
+If a snippet of Cypher can validly be a keyword as well as an unquoted identifier, it is always interpreted as a keyword.
```

Comment:
wondering about "snippet" - the term suggests that it can be more than one word, while "keyword" suggests that it's a single one (or rather a string which doesn't contain spaces, thinking of "ALL_SHORTEST_PATHS" etc)

would "expression" work?

Reply:
"expression" has a precise meaning. `a + b` is an expression to, while e.g. a label identifier alone is not an expression. We do not have a proper term, so I picked something non-technical. 

However, I am not married to "snippet". Not really a fan of my choice here in fact.

- A piece of Cypher 
- A part of Cypher
- Cypher text
- ...

any would work. 

Or a totally difference angle may be this:

> If an unquoted identifier equal to a keyword is user in Cypher in a place where can also the keyword, then it is interpreted as the keyword.

BTW: Feel free to edit the PR directly.